chainerx/_docs/routines.py

```diff
@@ -664,6 +664,46 @@ def _docs_linalg():
 .. seealso:: :func:`numpy.linalg.pinv`
 """)
 
+    _docs.set_doc(
+        chainerx.linalg.eigh,
+        """eigh(a, UPLO='L')
```

Comment:
Let's add a note to the docs that it's not backproppable, and that if one wants to, should use `eigvalsh`.

Reply:
Done. Actually it is the opposite. `eigvalsh` (computes only eigenvalues) is not backproppable because eigenvectors are required.